corehq/apps/settings/views.py

```diff
@@ -265,6 +265,7 @@ def post(self, request, *args, **kwargs):
 
             res = redirect(reverse(MyAccountSettingsView.urlname))
             res.set_cookie(settings.LANGUAGE_COOKIE_NAME, self.request.couch_user.language)
```

Comment:
can we not reuse this cookie rather than creating a new one?

Reply:
This is no longer applicable but this that is the [Django language cookie](https://docs.djangoproject.com/en/5.1/ref/settings/#language-cookie-name), and to prevent conflicts I opted for a separate cookie